src/compiler/types.ts

```diff
@@ -2590,9 +2590,16 @@ namespace ts {
         readonly parent: JsxAttributes;
         readonly name: Identifier;
         /// JSX attribute initializers are optional; <X y /> is sugar for <X y={true} />
-        readonly initializer?: StringLiteral | JsxExpression;
+        readonly initializer?: JSXAttributeValue;
     }
 
+    export type JSXAttributeValue =

```

Comment:
Should this be `JsxAttributeValue` to match the node types?

Reply:
@j-f1 That makes sense. Thanks.